src/main/java/com/ldtteam/structurize/util/LanguageHandler.java

```diff
@@ -32,86 +29,6 @@ private LanguageHandler()
         // Intentionally left empty.
     }
 
-    /**
-     * Send a message to the player.
-     *
-     * @param player  the player to send to.
-     * @param key     the key of the message.
-     * @param message the message to send.
-     */
-    public static void sendPlayerMessage(final Player player, final String key, final Object... message)
-    {
-        player.displayClientMessage(buildChatComponent(key.toLowerCase(Locale.US), message), false);
-    }
-
-    public static MutableComponent buildChatComponent(final String key, final Object... message)
-    {
-        MutableComponent translation = null;
-
-        int onlyArgsUntil = 0;
-        for (final Object object : message)
-        {
-            if (object instanceof Component)
-            {
-                if (onlyArgsUntil == 0)
-                {
-                    onlyArgsUntil = -1;
-                }
-                break;
-            }
-            onlyArgsUntil++;
-        }
-
-        if (onlyArgsUntil >= 0)
-        {
-            final Object[] args = new Object[onlyArgsUntil];
-            System.arraycopy(message, 0, args, 0, onlyArgsUntil);
-
-            translation = Component.translatable(key, args);
-        }
-
-        for (final Object object : message)
-        {
-            if (translation == null)
-            {
-                if (object instanceof Component)
-                {
-                    translation = Component.translatable(key);
-                }
-                else
-                {
-                    translation = Component.translatable(key, object);
-                    continue;
-                }
-            }
-
-            if (object instanceof Component)
-            {
-                translation.append(Component.empty());
-                translation.append((Component) object);
-            }
-            else if (object instanceof String)
-            {
-                boolean isInArgs = false;
-                for (final Object obj : translation.getSiblings())
-                {
-                    if (obj.equals(object))
-                    {
-                        isInArgs = true;
-                        break;
-                    }
-                }
-
-                if (!isInArgs)
-                {
-                    translation.append(" " + object);
-                }
-            }
-        }
```

Comment:
Will this cause an API issue with minecolonies?

Reply:
No, I checked, which is why I left the `translateKey*` methods, as those are all that mcol uses.  See the javadoc change at the top of the file 😄

Reply:
MineColonies is mostly using `MessageBuilder` now instead, it's only the config-file stuff that's still using `LanguageHandler`, and the comments on that indicate it doesn't really want to be.

Reply:
kk